examples/snos.yaml_template

```diff
@@ -0,0 +1,22 @@
+station:
```

Comment:
Short explanations would be nice for all the sections..

Reply:
Done